back/src/config/pg/kysely/model/database.ts

```diff
@@ -580,3 +581,9 @@ interface ImmersionAssessments {
   created_at: Generated<Timestamp | null>;
   updated_at: Generated<Timestamp | null>;
 }
+
+interface MarketingEstablishmentContacts {
+  siret: string;
+  email: string;
+  contact_history: Json;
```

Comment:
j'ai découvert récemment qu'il y a un utilitaire pour profiter d'un peu de typage sur les type JSONB:

```suggestion
  contact_history: JSONColumnType<TonTypePourCetteColonne>
```